### PR TITLE
Fix Aol icon and checkbox for login.yahoo.com

### PR DESCRIPTION
**Pull request by mstr3336**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2730,6 +2730,24 @@ td.c {
...
+}
+
+.social-login {
+    filter: invert(1) !important;
+}
+
+IGNORE IMAGE ANALYSIS
+.social-login
+
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2750]: Remove extra line

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -2730,6 +2730,24 @@ td.c {
...
+login.yahoo.com
+
+CSS
+label[for=persistent]::before {
+    background-color: ${black} !important;
+    filter: invert(1) !important;
+}
+
+.social-login {
+    filter: invert(1) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2744]: Why note using INVERT?

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -2730,6 +2730,24 @@ td.c {
...
 }
 
+================================
+
+login.yahoo.com
+
+CSS
+label[for=persistent]::before {
+    background-color: ${black} !important;
+    filter: invert(1) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2740]: Why not using INVERT?

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -2730,6 +2730,24 @@ td.c {
     color: ${white} !important;
 }
 
+================================
+
+login.yahoo.com
+
+CSS
+label[for=persistent]::before {
+    background-color: ${black} !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2739]: Invert and background-color?

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (mstr3336) [reply src/config/dynamic-theme-fixes.config:2740]: I'll try it, i think i landed here by trial and error w/ the selector, so I'll check that INVERT with that selector works

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (mstr3336) [reply src/config/dynamic-theme-fixes.config]: Resolved by ca3eead

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (mstr3336) [reply src/config/dynamic-theme-fixes.config]: Resolved by ca3eead

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (mstr3336) [reply src/config/dynamic-theme-fixes.config]: Resolved by ca3eead

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (mstr3336) [reply src/config/dynamic-theme-fixes.config]: Resolved by ca3eead

**Comment by Gusted** [comment]
Hi @mstr3336,

Where can I find the `.social-login` element?

**Comment by mstr3336** [comment]
> Hi @mstr3336,
> 
> Where can I find the `.social-login` element?

https://login.yahoo.com/?.src=finance&.intl=us&.lang=en-US&.done=https%3A%2F%2Ffinance.yahoo.com%2F

It was surprisingly tricky to find! When I first tried to find per your request, I couldn't, and it was only until I logged in on a new computer that it came up, coming from finance.yahoo.com.

**Comment by Gusted** [comment]
Tricky bastards.
Anyway LGTM

**Pull request merged**